pluginExamples/Exporters/SaveAsAngstrom.py

```diff
@@ -19,6 +19,7 @@ def __init__(self, ctl):
 
         # e.g. C:\Users\mzieg\Documents\EnlightenSpectra\SaveAsAngstrom
         # @todo could now use enlighten_info.get_save_path()
+        # SB 2023-11-02, actually should use common.get_default_data_dir() + "/SaveAsAngstrom"
```

Comment:
Note this is on 4.1.0-dev: https://github.com/WasatchPhotonics/ENLIGHTEN/commit/b5225979c83b03f95904d2e3dcbf634b09869b01#diff-ed2f9cf7613fca68b1fd4acf3cdb0c812c8171db655099d219c21ce2035a3a22R21